backend/plugins/frontline_api/src/modules/channel/graphql/resolvers/mutations/channel.ts

```
@@ -0,0 +1,145 @@
+export const channelMutations = {
+  channelAdd: async (
+    _parent: undefined,
+    {
+      name,
+      description,
+      icon,
+      memberIds,
+    }: { name: string; description: string; icon: string; memberIds: string[] },
+    { models, subdomain }: IContext,
+  ) => {
+    let user = { _id: '12' };
```

ESLint

[error] 'user' is never reassigned. Use 'const' instead.

(prefer-const)